Cargo.toml

```diff
@@ -25,10 +25,10 @@ members = [
     "module-system/sov-modules-stf-template",
     "module-system/sov-modules-rollup-template",
     "module-system/sov-modules-macros",
+    "module-system/sov-modules-core",
     "module-system/sov-state",
     "module-system/sov-modules-api",
     "module-system/module-schemas",
-    "module-system/utils/sov-first-read-last-write-cache",
```

Comment:
Do I understand corrrectly, that this crate merged into `sov-modules-core`

Reply:
Yes, that is correct. All the types there are required by `sov-modules-core`